src/capture/capture_midi.cpp

```diff
@@ -79,7 +80,7 @@ static void raw_midi_add_number(const uint32_t val)
 
 static void create_midi_file()
 {
-	midi.handle = CAPTURE_CreateFile("MIDI output", ".mid");
+	midi.handle = CAPTURE_CreateFile(CaptureType::Midi);
```

Comment:
very nice seeing the enum type being used instead of those mushy strings.

Reply:
Oh yes, 100%!